### ISSUE TITLE
Compatibility issue

### ISSUE THREAD

**Issue by Kazu967** [Madevil/KK_AccStateSync#6]
Plugin seem to be incompatible with Jali49 more accessories ( https://github.com/jalil49/MoreAccessories ), any chance of an update to make it compatible?

**Comment by EveKaczmarek** [comment]
> Plugin seem to be incompatible with Jali49 more accessories ( https://github.com/jalil49/MoreAccessories ), any chance of an update to make it compatible?

I built this plugin with experimental MoreAccessiories support. I do not recommend to use it but at least you can try for yourself.
https://github.com/EveKaczmarek/KK_AccStateSync/releases/tag/v4.5.0.0

**Comment by Kazu967** [comment]
> > Plugin seem to be incompatible with Jali49 more accessories ( https://github.com/jalil49/MoreAccessories ), any chance of an update to make it compatible?
> 
> I built this plugin with experimental MoreAccessiories support. I do not recommend to use it but at least you can try for yourself. https://github.com/EveKaczmarek/KK_AccStateSync/releases/tag/v4.5.0.0

Sorr for the late reply.
Not working, the plugin menu is not showing up. Didn't got any error when opening the game.

**Comment by EveKaczmarek** [comment]
> > > Plugin seem to be incompatible with Jali49 more accessories ( https://github.com/jalil49/MoreAccessories ), any chance of an update to make it compatible?
> > 
> > 
> > I built this plugin with experimental MoreAccessiories support. I do not recommend to use it but at least you can try for yourself. https://github.com/EveKaczmarek/KK_AccStateSync/releases/tag/v4.5.0.0
> 
> Sorry for the late reply. Seems to work, thanks! Is there also a KK_AAAPK version?

Yeah, I forgot about it. Check my profile for other builds, I'll push them soon.
